Fails to build with netcdf 4.9.3

I'm working on updating netcdf to 4.9.3 in Fedora.  netcdf-cxx4 4.3.1 fails to build with:
```
libtool: link: g++ -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -Wl,-z -Wl,relro -Wl,--as-needed -Wl,-z -Wl,pack-relative-relocs -Wl,-z -Wl,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -Wl,--build-id=sha1 -specs=/usr/lib/rpm/redhat/redhat-package-notes -o .libs/pres_temp_4D_plugin_rd pres_temp_4D_plugin_rd.o  ../cxx4/.libs/libnetcdf_c++4.so -lnetcdf
/usr/bin/ld: /tmp/ccRDJuAl.ltrans0.ltrans.o: in function `main':
/builddir/build/BUILD/netcdf-cxx4-4.3.1-build/netcdf-cxx4-4.3.1/build/examples/../../examples/pres_temp_4D_plugin_wr.cpp:74:(.text.startup+0x17b): undefined reference to `nc_set_log_level'
/usr/bin/ld: /builddir/build/BUILD/netcdf-cxx4-4.3.1-build/netcdf-cxx4-4.3.1/build/examples/../../examples/pres_temp_4D_plugin_wr.cpp:106:(.text.startup+0x2dd): undefined reference to `nc_set_log_level'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:722: pres_temp_4D_plugin_wr] Error 1
```

--- a/cxx4/test_filter.cpp
+++ b/cxx4/test_filter.cpp
@@ -30,7 +30,9 @@ int main()
   try
     {
       NcFile test("pres_temp_plugin_4D.nc", NcFile::replace);
+#ifdef HAVE_NC_SET_LOG_LEVEL
       nc_set_log_level(5);
+#endif
       // Define the dimensions. NetCDF will hand back an ncDim object for
       // each.
       NcDim latDim = test.addDim(LAT_NAME, NLAT);

--- a/examples/pres_temp_4D_plugin_wr.cpp
+++ b/examples/pres_temp_4D_plugin_wr.cpp
@@ -71,7 +71,9 @@ string LON_UNITS = "degrees_east";
 
 int main()
 {
+#ifdef HAVE_NC_SET_LOG_LEVEL
   nc_set_log_level(5);
+#endif
   // We will write latitude and longitude fields.
    float lats[NLAT],lons[NLON];
 
@@ -103,7 +105,9 @@ int main()
 
       // Create the file.
       NcFile test(FILE_NAME, NcFile::replace);
+#ifdef HAVE_NC_SET_LOG_LEVEL
       nc_set_log_level(5);
+#endif
       // Define the dimensions. NetCDF will hand back an ncDim object for
       // each.
       NcDim lvlDim = test.addDim(LVL_NAME, NLVL);

